lib/rules/require-default-props.js

```diff
@@ -105,6 +106,13 @@ module.exports = {
       if (node.type === 'Identifier') {
         return findVariableByName(node.name);
       }
+      if (
+        node.type === 'CallExpression' &&
+        propWrapperFunctions.has(node.callee.name) &&
+        node.arguments && node.arguments[0]
```

Comment:
It shouldn't matter, if `arguments` is empty, then `arguments[0]` will return undefined, which will cause this test to fail. It also handles the case where the first element of arguments contains something that is falsy :) 

It looks like checking the arguments array this way also matches code style in other places in this project.